lupa/tests/test.py

```diff
@@ -3009,27 +3009,33 @@ def load_tests(loader, standard_tests, pattern):
 class TestMaxMemory(SetupLuaRuntimeMixin, LupaTestCase):
     lua_runtime_kwargs = {"max_memory": 10000}
 
+    def setUp(self):
+        # need to test in here because the creation of the LuaRuntime fails
+        if "luajit" in self.lupa.LuaRuntime().lua_implementation.lower():
+            self.skipTest("not supported in LuaJIT")
+        return super().setUp()
```

Comment:
`super()` requires arguments in Py2.7.

Reply:
How long will lupa still support py2? It's been EOL for over 1.5 years.